CodeJam.Main/Reflection/AssemblyExtensions.cs

```diff
@@ -49,6 +50,18 @@ public static Stream GetRequiredResourceStream([NotNull] this Assembly assembly,
 		}
 
 #if TARGETS_NET || NETSTANDARD15_OR_GREATER || TARGETS_NETCOREAPP // PUBLIC_API_CHANGES
+		/// <summary>
```

Comment:
Is it really necessary to drag a new dependency for such a small thing?

Reply:
FileSystemInfo is a standard BCL thing. Sadly, netcore1 / netstd1 do not add reference to some standard libraries by default.